content/blog/2019-11-dtplyr-1-0-0/index.Rmarkdown

```diff
@@ -0,0 +1,228 @@
+---
+title: dtplyr 1.0.0
+author: Hadley Wickham
+date: '2019-11-08'
+slug: dtplyr-1-0-0
+description: |
+  A total rewrite of dtplyr is now available on CRAN; it now performs
+  computation lazily (like dbplyr), making it much more performant.
+categories:
+  - package
+tags:
+  - dtplyr
+  - dplyr
+  - tidverse
+photo:
+  url: https://unsplash.com/photos/h13Y8vyIXNU
+  author: Jay Ruzesky
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+I'm very excited to announce that [dtplyr](https://dtplyr.tidyverse.org) 1.0.0 is now on CRAN. dtplyr provides a [data.table](http://r-datatable.com/) backend for dplyr, allowing you to write dplyr code that is automatically translated to the equivalent, but usually much faster, data.table code.  
```

Comment:
> code that is automatically translated to the equivalent, but usually much faster, data.table code.

This resolves in a weird way where it's unclear if you're saying it's faster than data.table code or dplyr code. (I know it's the latter, but I guarantee it'll be parsed incorrectly). I think you can leave it as equivalent data.table code here, and leave the performance boost for later.